spec/unit/lcm/release_table_helper_spec.rb

```diff
@@ -82,5 +122,18 @@
         expect(GoodData::LCM2::Helpers.path_to_release_table_file(domain_id, data_prod_id, segment_id)).to eq 'release-tables/mydomain/myprod-mysegment.csv'
       end
     end
+
+    describe '.master_project_list_from_nfs' do
+      let(:result) do
+        [
+            { master_project_id: 'asdf12', version: 1 },
+            { master_project_id: 'ghjk34', version: 2 }
+        ]
+      end
+
+      it 'returns master project list' do
+        expect(GoodData::LCM2::Helpers.get_master_project_list_from_nfs(domain_id, data_prod_id, 'first_segment')).to eq(result)
+      end
+    end
   end
```

Comment:
Add test for update_master_project_from_nfs and remove_old_master_workspace

Reply:
updated it